examples/patient-hub/build.gradle

```diff
@@ -0,0 +1,36 @@
+buildscript {
```

Comment:
For all example projects going forward, I think we should use the following script. This defaults to a fixed version of DHF - which we can assume to be 5.0.2, as this will be part of the 5.0.2 release - and also allows for easily using 5.0-SNAPSHOT by setting -Ptesting=true on the command line or by adding testing=true to gradle.properties. 

Otherwise, a user can't easily make use of this Gradle file without publishing the DHF Gradle plugin locally. 

Note that this script is in dhf5-custom-hook and the smart-mastering-complete example project. 

```
buildscript {
    repositories {
        mavenLocal()
        maven { url "https://plugins.gradle.org/m2/" }
        jcenter()
    }

    dependencies {
        classpath "net.saliman:gradle-properties-plugin:1.5.1"
        if (project.hasProperty("testing")) {
            classpath "com.marklogic:ml-data-hub:5.0-SNAPSHOT"
        } else {
            classpath "gradle.plugin.com.marklogic:ml-data-hub:5.0.2"
        }
    }
}

apply plugin: "net.saliman.properties"
apply plugin: "com.marklogic.ml-data-hub"
```

Reply:
Excellent idea \m/